helm/ph-ee-engine/ams-paygops/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ph-ee-connector-ams-paygops-network-policy
+# From authorized namespaces
+  namespace: paymenthub   
+spec:
+  podSelector:
+    matchLabels:
+      name: ph-ee-connector-ams-paygops
+      app: ph-ee-connector-ams-paygops
+  policyTypes:
+  - Ingress  
+  - Egress
+# Allow inbound connections
+  ingress:   
+  - from:
+    - podSelector:
+        matchLabels:
+          app: ph-ee-connector-ams-paygops
+    ports:
+    - protocol: TCP
+      port: 8080
```

Comment:
if the port are hard coded, how will it make sure that the port will be the same as the service's port and the liveness readiness ports